mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -84,185 +90,200 @@ class MultipleFlightpathControlWidget(QtWidgets.QWidget, ui.Ui_MultipleViewWidge
     on the TopView canvas.
     """
 
-    def __init__(self, parent=None, view=None, listView=None, activeFlightTrack=None):
+    def __init__(self, parent=None, view=None, listView=None,
+                 listOperationsMSC=None, activeFlightTrack=None):
         super(MultipleFlightpathControlWidget, self).__init__(parent)
+        self.listOperationsMSC = listOperationsMSC
         self.listView = listView
         self.ui = parent
         self.setupUi(self)
         self.view = view  # canvas
         self.flight_path = None  # flightpath object
+        self.colour = None
```

Comment:
self.color?

Just seen that we used it different written all over MSS, on other places you use self.color.

Reply:
Removed